Dockerfile

```diff
@@ -3,8 +3,8 @@
 
 FROM alpine:3.18
 
-LABEL maintainer="OpenTF Team <opentf@opentf.org>"
+LABEL maintainer="OpenTofu Team <opentf@opentf.org>"
```

Comment:
Email left unchanged.

Reply:
we don't have a new mail yet
waits for LF